components/collector/src/source_collectors/jira/issue_status.py

```diff
@@ -23,7 +27,16 @@ async def _parse_issue_status(self, responses: SourceResponses) -> IssueStatus:
         """Override to get the issue status from the responses."""
         json = await responses[0].json()
         name = json["fields"]["status"]["name"]
+        jira_status_category = json["fields"]["status"]["statusCategory"]["key"]
```

Comment:
The status category should always there. But, if it's not, the user will get an error message in the UI. That's because the collector does all of its work within a broad exception handler that makes sure it doesn't crash. If an exception happens, the error is added to the `IssueStatus` instance and reported to the user.